templates/download/desktop/thank-you.html

```diff
@@ -40,240 +41,328 @@ <h1 style="font-weight: 100;">Thank you for downloading Ubuntu Desktop</h1>
+          <h3 class="p-heading--5"><a href="/tutorials/how-to-run-ubuntu-desktop-on-a-virtual-machine-using-virtualbox">How to run Ubuntu Desktop using VirtualBox</a></h3>
+          <p>Run Ubuntu Desktop in a virtual machine using VirtualBox. A quick start guide that will work across any operating system.</p>
+        </div>
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5"><a href="/tutorials/how-to-install-ubuntu-desktop-on-raspberry-pi-4">How to install Ubuntu Desktop on Raspberry Pi 4</a></h3>
+          <p>A complete guide to installing Ubuntu Desktop on a Raspberry Pi 4 (2GB or above).</p>
+        </div>
+      </div>
     </div>
-    <div class="col-4 p-divider__block">
-      <h3 class="p-heading--4">Help is always at hand</h3>
-      <p>Join our global community for support and tips on how to get started with Ubuntu.</p>
-      <ul class="p-list">
-          <li><a href="https://discourse.ubuntu.com/">Ubuntu Discourse&nbsp;&rsaquo;</a></li>
-          <li><a href="https://askubuntu.com/">Ask Ubuntu&nbsp;&rsaquo;</a></li>
-          <li><a href="https://answers.launchpad.net/ubuntu">Launchpad Answers&nbsp;&rsaquo;</a></li>
-          <li><a href="https://wiki.ubuntu.com/IRC/ChannelList">IRC-based support&nbsp;&rsaquo;</a></li>
+    <div class="col-3">
+      <div class="p-section--shallow">
+        <h2 class="p-muted-heading">Help is always at hand</h2>
+      </div>
+      <ul class="p-list--divided">
+        <li class="p-list__item"><a href="https://help.ubuntu.com/stable/ubuntu-help">Ubuntu documentation</a></li>
+        <li class="p-list__item"><a href="https://discourse.ubuntu.com/">Ubuntu Discourse</a></li>
+        <li class="p-list__item"><a href="https://askubuntu.com/">Ask Ubuntu</a></li>
+        <li class="p-list__item"><a href="https://answers.launchpad.net/ubuntu">Launchpad Answers</a></li>
       </ul>
     </div>
   </div>
 </section>
 
-<section class="p-strip--light">
-  {% include "shared/_pro_strip.html" %}
+<section class="p-section">
   <div class="u-fixed-width">
-    <p>For more information, download our whitepaper:</p>
-    <p><a href="/engage/adopting-secure-enterprise-linux-desktop" class="p-button">Adopting a secure enterprise Linux desktop</a></p>
+    <a href="/pro">
+    {{
+      image (
+      url="https://assets.ubuntu.com/v1/2f32b5f5-Ubuntu-Pro-logo.svg",
+      alt="Ubuntu Pro",
+      width="170",
+      height="42",
+      hi_def=True,
+      loading="lazy"
+      ) | safe
+    }}
+    </a>
+    <hr class="p-rule" />
   </div>
-  <div class="u-fixed-width">
-    <p><a class="p-button--positive" href="/pro">Register today</a></p>
+  <div class="row--50-50">
+    <div class="col">
+      {% if page == "desktop" %}
+        <h2>Secure enterprise management<br class="u-hide--medium u-hide--small"/> with Ubuntu Pro Desktop</h2>
+      {% else %}
+        <h2>Get enterprise support<br class="u-hide--medium u-hide--small"/> for Ubuntu Server</h2>
+      {% endif %}
```

Comment:
This if block isn't working it seems 
![image](https://github.com/canonical/ubuntu.com/assets/17607612/c6b5d1fe-137e-48bc-9b64-a0faa679f2ae)